tests/core/test_kontainer_drivers.py

```diff
@@ -42,6 +42,10 @@ def test_kontainer_driver_lifecycle(admin_mc, remove_resource):
     # type are removed from the name
     assert kd.name == "example"
 
+    # verify the kontainer driver has activate and deactivate links
+    assert kd.actions.activate is not None
```

Comment:
I don't think checking for None is enough as an empty string will pass this check

Reply:
Changed to != "" check.